addons/auth_totp_portal/static/src/scss/auth_totp_portal.scss

```diff
@@ -0,0 +1,31 @@
+.o_auth_totp_enable_2FA {
```

Comment:
Mmmh ... to remove as much as possible (either bootstrap, either "standard form view ballek").

Reply:
The issue here is that we have to adapt the backend view into frontend environment. I just took the style from o_field_copy from backend to use it in our weirdo case in frontend usage of the wizard.
I could eventually import the css file that includes the o_field_copy class in frontend assets, but I would import a lot more than just what we need. Soo.. meh'
And I can't just put the bootstrapable styles into the original backend view as o_field_copy is a generated class from the copyclipboard widget. Unfortunatelly, I think we have no choice here. UGLY, but consistent with the rest of this module's mess.
EDIT: I'll add a note in the file to explain that

Reply:
Ok, note is fine until we figure out an easier way to include backend stuff in frontend (aka: never).